DSESwift/DataPoint.swift

```diff
@@ -55,7 +55,7 @@ public class DataPoint{
      * Set time of the data point.
      * @param time: NSDate for the timestamp of the datapoint.
      **/
-    func setTime(time: NSDate){
+    public func setTimeWithNSDate(time: NSDate){
```

Comment:
@kanekin introducing another function 'setTime' and make the setTimeWithNSDate to just forward the call to it. But you already explain that the framework have already reserver the 'setTime' function, so i think it's impossible now. Anyway, is this 'setTime' function is callable from swift?